### PR TITLE
Fix help text for new command

### PR DESCRIPTION
**Pull request by suganyasuven**
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/29126

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help [hunk 1]
```diff
@@ -46,9 +46,9 @@ DESCRIPTION
             Create a package with a 'main' template.
                $ bal new <package-name>
 
-            Create a package with a Ballerina service.
+            Create a package with a ballerina service template
```
Comment (pramodya1994) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help:49]: This doesn't have a `.` in the end, But L 52 has.

Comment (NipunaRanasinghe) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help:49]:
```suggestion
            Create a package with a Ballerina service template.
```

**Review by pramodya1994 — CHANGES_REQUESTED** [review]
Pls check the comment

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help [hunk 2]
```diff
@@ -46,9 +46,9 @@ DESCRIPTION
...
                $ bal new <package-name> -t service
 
-            Create a package with a Ballerina service.
+            Create a package with a ballerina library template.
```
Comment (NipunaRanasinghe) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help:52]:
```suggestion
            Create a package with a Ballerina library template.
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help [hunk 3]
```diff
@@ -43,12 +43,12 @@ DESCRIPTION
                 Create a package using a predefined template.
 
        EXAMPLES
-            Create a package with a 'main' template.
+            Create a package with a Ballerina main template.
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help:46]:
```suggestion
            Create a package with a 'main function.
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help [hunk 4]
```diff
@@ -43,12 +43,12 @@ DESCRIPTION
...
                $ bal new <package-name>
 
-            Create a package with a Ballerina service.
+            Create a package with a Ballerina service template.
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help:49]: I think the previous one is fine. We are creating a file with a service

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help [hunk 5]
```diff
@@ -43,12 +43,12 @@ DESCRIPTION
...
                $ bal new <package-name> -t service
 
-            Create a package with a Ballerina service.
+            Create a package with a Ballerina library template.
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help:52]:
```suggestion
            Create a package with the Ballerina library template.
```

**Review by azinneera — APPROVED** [review]
LGTM

**Review by pramodya1994 — APPROVED** [review]

**Pull request merged**